modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -11,11 +11,45 @@
 
 enum { DIRBIT = 1, FILEBIT = 2 };
 
+static BYTE GetPIDLType(LPCITEMIDLIST pidl)
+{
+    return pidl && pidl->mkid.cb >= 3 ? (pidl->mkid.abID[0] & 0x7F) : 0;
+}
+
+struct FS95
+{
+    WORD cb;
+    BYTE type;
+    BYTE unknown;
+    UINT size;
+    WORD date, time;
+    WORD att;
+    CHAR name[ANYSIZE_ARRAY];
```

Comment:
Just to be sure, this is standard layout of common PIDL?

Reply:
It is the layout of Windows95 "CFSFolder" items and the header for later versions.